sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -74,10 +81,36 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
           "& :where(a strong):not(:where([class~='not-prose'] *))": {
             color: safeSiteColor,
           },
+
+          px: '$4',
+
+          '& p': {
+            m: 0,
+            p: 0,
+          },
         }}
       >
-        <h1 className="sigle-title">{story.title}</h1>
-        {story.coverImage && (
+        <Link href="/[username]" as={`/${username}`} passHref>
+          <a>
+            <Text size="action">{siteName}</Text>
+            <Box css={{ mt: '$1', mb: '$7' }}>
```

Comment:
is the box necessary around the Text? probably the margin can be added to the text to avoid using box :)